modules/i3s/src/lib/parsers/parse-i3s-tile-content.js

```diff
@@ -146,8 +148,9 @@ async function parseI3SNodeGeometry(arrayBuffer, tile = {}, options) {
   content.attributes = {
     positions: attributes.position,
     normals: attributes.normal,
-    colors: normalizeColors(attributes.color),
+    colors: normalizeAttribute(attributes.color), // Normalize from UInt8
     texCoords: attributes.uv0,
+    uvRegions: normalizeAttribute(attributes.uvRegion), // Normalize from UInt16
```

Comment:
You could just use the luma.gl feature to set this to a constant value (single array) when no region is available.